cfgov/templates/wagtailadmin/table_input.html

```diff
@@ -69,9 +69,47 @@
         </div>
     </div>
 </div>
+<div class="field boolean_field widget-checkbox_input">
+    <label for="{{ attrs.id }}-handsontable-sortable">Sortable table</label>
+    <div class="field-content">
+        <div class="input">
+            <input type="checkbox" id="{{ attrs.id }}-handsontable-sortable" name="handsontable-sortable"/>
+            <span class="help">( Enable sortable table functionality )</span>
+        </div>
+    </div>
+</div>
 <br/>
```

Comment:
Since we're here, let's get rid of this unnecessary self-closing slash and the ones on all `input`s above.